packages/flutter/test/widgets/text_golden_test.dart

```diff
@@ -242,10 +240,9 @@ void main() {
     );
     await expectLater(
       find.byType(Container),
-      matchesGoldenFile('text_golden.StrutDefault.png'),
+      matchesSkiaGoldFile('text_golden.StrutDefault.png'),
     );
-  }, skip: true); // Should only be on linux (skip: !Platform.isLinux).
-                  // Disabled for now until font inconsistency is resolved.
+  },);
```

Comment:
Does this resolve font inconsistencies the original one mentioned?

Reply:
I am not certain yet. I wanted to try to test it across platforms and see if it is no longer an issue due the fact that Skia Gold manages all potential variations of a golden file. In this system there is no singular, true golden master, but several.